dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -2052,6 +2052,26 @@ def assert_bucket_hard_deletes_contents(bucket)
     assert teacher_old.purged_at
   end
 
+  test 'deletes contact rollups final data' do
+    teacher = create :teacher
+    teacher_email = teacher.email
+    create :contact_rollups_final, email: teacher_email
+
+    assert ContactRollupsFinal.find_by_email(teacher_email)
+    purge_user teacher
+    assert_nil ContactRollupsFinal.find_by_email(teacher_email)
```

Comment:
Do you also need the `.with_deleted` so it checks for deleted data?

Reply:
ContactRollupsFinal and ContactRollupsPardotMemory don't use paranoia so they don't have `.with_deleted`.